### PR TITLE
Added RGB compatible FC660C keymaps. Added VIA support and fixed indicators on Cajal keymaps

### PR DESCRIPTION
**Pull request by rooski15**
Added two keymaps to FC660C, both replacing insert LED with RGB control.
Modified default and default_ortho keymaps in Cajal to fix dim indicator LEDs.

## Description
Developed a mod to add RGB to FC660C by replacing Insert indicator with RGB. Copied 'default' and 'via' keymap folders, and added keymap specific config.h and rules.mk.

Restored LED indicator code found in WSK original keymaps to enable full-brightness indicator LEDs on Cajal keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Walletburner Cajal LEDs run at half-brightness. Due to missing code to set pins as output.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/fc660c/keymaps/via_rgb/fc660c_via_RGB.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (drashna) [inline keyboards/fc660c/keymaps/via_rgb/fc660c_via_RGB.json:1]: This is a VIA json and doesn't belong in this repo.   Please remove it.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/fc660c/keymaps/via_rgb/fc660c_via_RGB.json (hunk 1 above)
Comment (rooski15) [reply keyboards/fc660c/keymaps/via_rgb/fc660c_via_RGB.json]: Done.

**Review by drashna — APPROVED** [review]

File: keyboards/walletburner/cajal/rules.mk [hunk 2]
```diff
@@ -18,3 +18,4 @@ BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
 ENCODER_ENABLE = yes    # Enable support for EC11 Rotary Encoder
+VIA_ENABLE = yes
```
Comment (drashna) [inline keyboards/walletburner/cajal/rules.mk:21]: please remove.  This should only be defined at the keymap level. 
```suggestion
```

Comment (rooski15) [reply keyboards/walletburner/cajal/rules.mk]: Understood

File: keyboards/fc660c/keymaps/default_rgb/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Keymap with Insert LED remapped to RGB control
```
Comment (drashna) [inline keyboards/fc660c/keymaps/default_rgb/README.md:1]: Please rename file this to be lower case.

File: keyboards/walletburner/cajal/keymaps/via_ortho/keymap.c [hunk 4]
```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 Worldspawn <mcmancuso@gmail.com>
```
Comment (drashna) [inline keyboards/walletburner/cajal/keymaps/via_ortho/keymap.c:1]: This keymap should be removed. 

Generally, you shouldn't need or have multiple via keymaps, as via should be able to handle different layouts in it's UI

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/walletburner/cajal/keymaps/via_ortho/keymap.c (hunk 4 above)
Comment (rooski15) [reply keyboards/walletburner/cajal/keymaps/via_ortho/keymap.c]: Good point - thank you.

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**